### PR TITLE
BRS-1127: Updated text bugs on site metrics page (#286)

### PR DESCRIPTION
**Pull request by simonlarocque**
### Description: Was this the smallest amount of code I could change? Yes. Did it take more than an entire day. You betcha. Am I stoked? HELL YA! #progressnotperfection

Changed two text fields on site metrics for DUP Admin. Tested on local environment.

### CODE REVIEW

**Review by meredithom — APPROVED** [review]
w nit

I know it's what's in the ticket, but it should probably be "choose", not "chose"

**Comment by simonlarocque** [comment]
> w nit
> 
> I know it's what's in the ticket, but it should probably be "choose", not "chose"

Okay should I change quickly? Should I submit a new PR?

**Comment by meredithom** [comment]
> > w nit
> > I know it's what's in the ticket, but it should probably be "choose", not "chose"
> 
> Okay should I change quickly? Should I submit a new PR?

No need for a new PR, you can just update this one.

File: src/app/shared/components/ds-forms/shortdate-picker/shortdate-picker.component.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
     [class.with-item-left]="icon"
     [disabled]="checkDisabled()"
     bsDatepicker
-    placeholder="Select date"
+    placeholder="Select date range"
```
Comment (cameronpettit) [inline src/app/shared/components/ds-forms/shortdate-picker/shortdate-picker.component.html:22]: Remove `range` here.  This section of code renders a non-range datepicker.

**Review by cameronpettit — CHANGES_REQUESTED** [review]

File: src/app/shared/components/ds-forms/shortdate-picker/shortdate-picker.component.html (hunk 1 above)
Comment (simonlarocque) [reply src/app/shared/components/ds-forms/shortdate-picker/shortdate-picker.component.html:22]: The other one on line 38 is okay?

Comment (cameronpettit) [reply src/app/shared/components/ds-forms/shortdate-picker/shortdate-picker.component.html:22]: Yessir

**Review by cameronpettit — APPROVED** [review]

**Pull request merged**